app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- While you've set up your devise correctly, users don't seem to get any confirmation email sent to their mails, so they can't log in to your app this is because there are some extra configurations you'd have to make to the mailer so they can get the confirmation, however, this is beyond the scope of your project now, otherwise you want to configure it. Here are a few things you should do to make users access your app. Please, follow closely:

   - Remove the `:confirmable`
   - this line should now be `:recoverable, :rememberable, :validatable`